service/src/main/java/bio/terra/workspace/service/policy/flight/MergePolicyAttributesDryRunStep.java

```diff
@@ -49,7 +49,11 @@ public StepResult doStep(FlightContext flightContext)
     }
 
     TpsPaoUpdateResult dryRunResults =
-        tpsApiDispatch.mergePao(destinationWorkspaceId, sourceWorkspaceId, TpsUpdateMode.DRY_RUN);
+        (cloningInstructions == CloningInstructions.LINK_REFERENCE)
+            ? tpsApiDispatch.linkPao(
+                destinationWorkspaceId, sourceWorkspaceId, TpsUpdateMode.DRY_RUN)
+            : tpsApiDispatch.mergePao(
```

Comment:
I think the mergePao arguments are reversed, it looks like the function expects (destination, source, updateMode)